From 2c2c91886d41b33c93e884a589a0444c28519b73 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 11 Sep 2024 17:07:11 +0200
Subject: [PATCH 1216/1695] keys: Add a null AES 32 bytes key for remoteproc
 encrypted firmware test

The key is aligned with the test key defined in
core/pta/stm32mp/remoteproc_pta.c
to use it the CFG_REMOTEPROC_ENC_TEST should be enable
the key can be provided to the sign_rproc_fw.py script using
'--enc_key' argument.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git eebb65d62d019ddfb04772480d7591f2add0323e keys: Add a null AES 32 bytes key for remoteproc encrypted firmware test]
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I29fbdef67ae897107213a79556b7dc94ecb7bdfd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/403807
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 keys/rproc_enc_test_key.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 keys/rproc_enc_test_key.txt

diff --git a/keys/rproc_enc_test_key.txt b/keys/rproc_enc_test_key.txt
new file mode 100644
index 000000000..cd09bbf16
--- /dev/null
+++ b/keys/rproc_enc_test_key.txt
@@ -0,0 +1 @@
+0000000000000000000000000000000000000000000000000000000000000000
-- 
2.47.2

